### DISCUSSION TITLE
What do you need for character support?

### DISCUSSION THREAD

**Discussion by jpsollie** [DiaKEM/dexcom-g7-apk-patcher#59] [Q&A]
Hi there,
I am trying to rewrite the patches for dexcom 2.x
The problemen I'm having is that my PC seems to handle special characters incorrectly, making it very hard to patch
Does anybody know what need to be set to support these special characters? I tried to run a Ubuntu vm, but didn't work either
Thanks

**Comment by mysteriouslyblah** [comment]
These invalid characters are a way of code obfuscation and are used intentionally to make working with the disassembled code difficult. I am using WSL and was able to navigate around this issue by using a python script to find the files I needed to edit and then using vim to edit them.
